CheapShopWeb/Identity/AppUserManager.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Web;
+using CheapShopWeb.DataContext;
+using CheapShopWeb.Models;
+using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.AspNet.Identity.Owin;
+using Microsoft.Owin;
+using Microsoft.Owin.Security;
+
+namespace CheapShopWeb
+{
+    public class AppUserManager : UserManager<User>
+    {
+        public AppUserManager(IUserStore<User> store)
+            : base(store)
+        {
+        }
+
+        // this method is called by Owin therefore this is the best place to configure your User Manager
+        public static AppUserManager Create(
+            IdentityFactoryOptions<AppUserManager> options, IOwinContext context)
+        {
+            var manager = new AppUserManager(new UserStore<User>(context.Get<MyDbContext>()));
+            
+            // Configure validation logic for usernames
+            manager.UserValidator = new UserValidator<User>(manager)
+            {
+                AllowOnlyAlphanumericUserNames = false,
+                RequireUniqueEmail = true
```

Comment:
Nelabai randu kur pachakine ar email unique? ar cia jis pats susitvarko ir susiziuri?

Reply:
Čia viską sutvarko identity user'is, ir man atrodo būtent šitoj vietoj ir yra nurodoma kad email'ai privalo buti unikalūs